main.go

```diff
@@ -33,6 +33,10 @@ func skipInit() bool {
 		return false
 	}
 	switch os.Args[1] {
+	case "--version":
+		return true
+	case "--help":
```

Comment:
```suggestion
	case "--help", "help":
```
Probably worth adding the non-flag variant as well.

Reply:
Good point.
It seems that "version" also has a 'non-flag' variant as well. I just pushed a new version of the patch with both added.

Reply:
Changes LGTM.